opus/application/apps/ui/views.py

```diff
@@ -505,9 +505,17 @@ def api_init_detail_page(request, **kwargs):
     new_products = OrderedDict()
     for version in products:
         new_products[version] = OrderedDict()
+        selection = None
         for product_type in products[version]:
             file_list = products[version][product_type]
             product_info = {}
+            if (selection is None and (product_type[3].find('Browse Image') or
+                product_type[3].find('Browse Diagram'))):
+                for fn in file_list:
```

Comment:
This seems unnecessarily complicated. Since you always get the first filename no matter what, can't you just do:

```
if len(file_list) > 0:
    selection = file_list[0].split('/')[-1].split('.')[0]
```